app/models/ability.rb

```diff
@@ -62,4 +66,10 @@ def custom_permissions
     def curation_concerns
       Hyrax.config.curation_concerns
     end
+
+    def download_file_permission
+      can :read, ActiveFedora::File do |file|
+        file.mime_type == 'image/tiff' ? false : true
```

Comment:
I see. So is this the primarily place where we're ensuring `original_file`'s aren't downloaded by non-curators?

Because if so, it seems like we have far more file types than `tiff` which would fall under this check. My reading of Ryan's comment is tiff was just a (common) example. Not the sole use case or instance where we have this use case. And, it also seems like it would be nice if we could check the `file_use` instead of the `mime_type` to determine download-ability? Should we clarify with Ryan?